### ISSUE TITLE
Passcode FAQ

### ISSUE THREAD

**Issue by thedoublejay** [DeFiCh/wallet#812]

**Comment by kyleleow** [comment]
@thedoublejay content is pending on content team and @jtlntno-cake is occupied on other work right now to finalize the placement of each faq page

**Comment by kyleleow** [comment]
@jtlntno-cake the current passcode creation screen is auto validation so the "create passcode" button is removed. By referring to your latest prototype, do you want to add it back? 

Current passcode creation:
![image](https://user-images.githubusercontent.com/40191153/133194721-9306991b-da57-4f13-adee-aa7eb51b9911.png)

**Comment by thedoublejay** [comment]
@kyleleow keep everything as is, just need to add the FAQ link from here

**Comment by izzycsy** [comment]
`Acceptance Criteria`
Expected: 

**Android:**

- [ ] `CREATE A WALLET` > `SHOW 24-WORDS` > Display Recovery Words > Verify > On `Secure with passcode` screen, select `Learn more about passcodes` > redirect to article on passcodes


**iOS:**

- [ ] `CREATE A WALLET` > `SHOW 24-WORDS` > Display Recovery Words > Verify > On `Secure with passcode` screen, select `Learn more about passcodes` > redirect to article on passcodes

**Comment by jtlntno-cake** [comment]
@kyleleow good call. no need to retain button if there's no use to it then. No need to add it back. Thanks!
